grammar.js

```diff
@@ -296,17 +353,32 @@ module.exports = grammar({
     )),
 
     pointer_declarator: $ => prec.dynamic(1, prec.right(seq(
+      optional($.ms_modifiers_based),
       '*',
+      optional($.ms_modifiers_unaligned_ptr),
+      optional($.ms_modifiers_restrict),
+      optional($.ms_modifiers_unsigned_ptr),
+      optional($.ms_modifiers_signed_ptr),
```

Comment:
This is another area where I think it's worth "loosening" things a bit. In terms of state machine size, it's easier to match things like `repeat(thing)` than it is to match `optional(thing1), optional(thing2), optional(thing3)...`.

What do you think of replacing these four optional elements with just `repeat($.ms_pointer_modifier)`, where `ms_pointer_modifier` would be a choice between any of the pointer-related keywords above?

Reply:
Works for me. I'll amend this.